pcdsdevices/epics_motor.py

```diff
@@ -1562,6 +1562,20 @@ class SmarActOpenLoop(Device):
     module_temp = Cpt(EpicsSignalRO, ':MODTEMP', kind='normal',
                       doc='Temperature of the MCS2 Module in the rack')
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Long name shenanigans
+        self.step_voltage.long_name = 'Step Voltage'
```

Comment:
side note: the long_name ophyd PR churned a little bit recently (rebase?) so these are pretty safe for the future.